src/Find-SDC-Repositories.js

```diff
@@ -0,0 +1,119 @@
+const graphql = require("graphql-request");
+const { zipWith } = require("lodash");
+const prompt = require("prompt");
+const fs = require("fs");
+const process = require("process");
+const mkdirp = require("mkdirp");
+
+const config = {
+  file: {
+    filePrefix: "SDC-Teams-Github-Repos-",
+    directoryPath: "outputs/SDCTeamGithubRepos/"
+  },
+  api: {
+    endpoint: "https://api.github.com/graphql",
+    options: {
+      headers: {
+        Authorization: null
+      }
+    },
+    query: `
+    query { 
+        organization(login:"ONSDigital") { 
+            teams(first: 100) {
+                nodes {
+                    name
+                    repositories(first: 100) {
+                        nodes {
+                            name
+                        }
+                        edges {
+                            permission
+                        }
+                    }
+                }
+            }
+        }
+    }`
+  }
+};
+
+prompt.start();
+
+prompt.get(["key"], function(err, result) {
+  config.api.options.headers.Authorization = "Bearer " + result.key;
+
+  if (
+    config.api.options.headers.Authorization !== null &&
+    config.api.options.headers.Authorization !== ""
+  ) {
+    mkdirp(config.file.directoryPath);
+
+    const client = new graphql.GraphQLClient(
+      config.api.endpoint,
+      config.api.options
+    );
+
+    client
+      .request(config.api.query)
+      .then(function(data) {
+        let output = { teams: [] };
+
+        data.organization.teams.nodes.forEach(team => {
+          let teamData = {};
```

Comment:
We're not re-assigining teamData within this block so it can be declared `const`.